backend/vxlan/vxlan.go

```diff
@@ -55,6 +59,10 @@ func NewOverlay(configDir string) (*Overlay, error) {
 		return nil, nil
 	}
 
+	if bridgeName != "" {
+		dockerBridgeName = bridgeName
```

Comment:
Can you please test this code by not using "docker0", some other bridge and see if things work?